File: app/src/main/java/com/enhance/enhancedemo/model/Fairings.java
package com.enhance.enhancedemo.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Fairings {

    @SerializedName("reused")
    @Expose
    private Object reused;
    @SerializedName("recovery_attempt")
    @Expose
    private Object recoveryAttempt;
    @SerializedName("recovered")
    @Expose
    private Object recovered;
    @SerializedName("ships")
    @Expose
    private List<String> ships = null;

    public Object getReused() {
        return reused;
    }

    public void setReused(Object reused) {
        this.reused = reused;
    }

    public Object getRecoveryAttempt() {
        return recoveryAttempt;
    }

    public void setRecoveryAttempt(Object recoveryAttempt) {
        this.recoveryAttempt = recoveryAttempt;
    }

    public Object getRecovered() {
        return recovered;
    }

    public void setRecovered(Object recovered) {
        this.recovered = recovered;
    }

    public List<String> getShips() {
        return ships;
    }

    public void setShips(List<String> ships) {
        this.ships = ships;
    }

}

File: app/src/main/java/com/enhance/enhancedemo/model/Fairings.kt
package com.enhance.enhancedemo.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Fairings {

    @SerializedName("reused")
    @Expose
    var reused: Any? = null
    @SerializedName("recovery_attempt")
    @Expose
    var recoveryAttempt: Any? = null
    @SerializedName("recovered")
    @Expose
    var recovered: Any? = null
    @SerializedName("ships")
    @Expose
    var ships: List<String>? = null

}